test/library/standard/DateTime/strptime2.future

```diff
@@ -0,0 +1,7 @@
+unimplemented feature: strptime() doesn't support '%f' flag
+
+#16922
+
+This version of the test uses a variant signature which was
+implemented in #17453, though I was unable to find any design
+discussion about the tradeoffs between the two approaches.
```

Comment:
It took me a while to figure out what this comment was about. Could you expand on the "variant signature" comment to say something like "This version of the test uses a mutating method instead of a type method to implement strptime. This was implemented in ..."